Speech to Text/MicrophoneAdvancedViewController.swift

```diff
@@ -37,8 +37,7 @@ class MicrophoneAdvancedViewController: UIViewController {
 
         // use `SpeechToTextSession` for advanced configuration
         session = SpeechToTextSession(
-            username: Credentials.SpeechToTextUsername,
-            password: Credentials.SpeechToTextPassword
+            apiKey: Credentials.SpeechToTextApiKey
```

Comment:
Can we add something in the README describing what to do if you have `username` and `password`?

Users should be able to specify `apikey` as username and the `<apikey value>` as password

Reply:
Maybe we can revert this change and add a few sentences in the README for users with an `apikey` rather than `username` and `password`

Reply:
Hi @germanattanasio, my reply comes a bit late because I have tested over and over again and if I initialize the `SpeechToTextSession` (in `MicrophoneAdvancedViewController.swift`) using username and password (using `apikey` as username), the transcription session does not work (although using `apikey` as username works for all the other APIs, such as SpeechToText). 

It works if I use the following on my iPhone 7+ running iOS 12.1.1 : 

```
session = SpeechToTextSession(
            apiKey: "<the api key>"
        )
```

That's the reason why I moved the authentication to using `apiKey` only, as I suspect the method accepting username and password has some issues. 

Please let me know if you find the time to verify, or perhaps it's worth filing a issue report to https://github.com/watson-developer-cloud/swift-sdk ?

Reply:
@germanattanasio I think we should leave it as is (with only the `apiKey` parameter) since that's the "standard" that we're trying to get new users to use. I agree that we should update the README too to give an example using `username` and `password` like we do in the swift-sdk README.

Reply:
@maggix I just checked the source code for the Watson Swift SDK, and I think you did find a bug. Can you open an issue in that repo?

Reply:
Done , https://github.com/watson-developer-cloud/swift-sdk/issues/950